JpaLayer/src/main/java/com/axinalis/config/KafkaConsumersConfiguration.java

```diff
@@ -0,0 +1,52 @@
+package com.axinalis.config;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.kafka.annotation.EnableKafka;
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+import org.springframework.kafka.support.serializer.JsonDeserializer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.springframework.kafka.support.KafkaHeaders.GROUP_ID;
+
+@Configuration
+@EnableKafka
+public class KafkaConsumersConfiguration {
+
+    @Value("${kafka.broker}")
+    private String KAFKA_BROKER;
+
+    @Bean
+    public ConsumerFactory<String, String> consumerFactory(){
+        return new DefaultKafkaConsumerFactory<>(consumerConfiguration());
+    }
+
+    @Primary
+    @Bean
+    public Map<String, Object> consumerConfiguration(){
+        Map<String, Object> configuration = new HashMap<>();
+
+        configuration.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, KAFKA_BROKER);
+        configuration.put(ConsumerConfig.GROUP_ID_CONFIG, GROUP_ID);
+        configuration.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, JsonDeserializer.class);
+        configuration.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, JsonDeserializer.class);
+
+        return configuration;
+    }
+
+    @Bean
+    ConcurrentKafkaListenerContainerFactory<String, String> kafkaListenerContainerFactory(){
+        ConcurrentKafkaListenerContainerFactory<String, String> factory =
+                new ConcurrentKafkaListenerContainerFactory<>();
+        factory.setConsumerFactory(consumerFactory());
+        return factory;
+    }
+
```

Comment:
you even may specify custom Kafka templates with predefined Serializing / Deserializing:
https://www.baeldung.com/spring-kafka